mscore/prefsdialog.cpp

```diff
@@ -959,133 +1128,148 @@ void PreferenceDialog::buttonBoxClicked(QAbstractButton* button)
             }
       }
 
+//---------------------------------------------------------
+//   applySetActive
+//---------------------------------------------------------
+
+void PreferenceDialog::applySetActive(bool active)
+      {
+      buttonBox->button(QDialogButtonBox::Apply)->setEnabled(active);
+      }
+
+
+//---------------------------------------------------------
+//   applyActivate
+//---------------------------------------------------------
+
+void PreferenceDialog::applyActivate()
+      {
+      applySetActive(true);
+      }
+
+//---------------------------------------------------------
+//   widgetModified
+//---------------------------------------------------------
+
+void PreferenceDialog::widgetModified()
+      {
+      // πρέπει να βλέπω αν αλλαξε τιμη απο την αρχικη ή αν επεστρεψε πισω στην αρχικη τιμη!!!
```

Comment:
hahaha I guess you forgot to remove or translate it.
I love this alphabet. It looks like a big math problem.

Reply:
Looks Greek to me ;-)

Reply:
:rofl: it's a reminder to think about implementing a smarter way to determine if a value has been modified. If the user changes all widgets back to their original values (before applying), apply should not be enabled.